OpenRA.Mods.Common/Lint/CheckTargetHealthRadius.cs

```diff
@@ -55,13 +55,13 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Ruleset ru
 					var missile = weaponInfo.Value.Projectile as MissileInfo;
 					var areabeam = weaponInfo.Value.Projectile as AreaBeamInfo;
 
-					if (bullet == null && missile == null && areabeam == null)
-						continue;
-
 					var targetExtraSearchRadius = bullet != null ? bullet.TargetExtraSearchRadius :
 						missile != null ? missile.TargetExtraSearchRadius :
 						areabeam != null ? areabeam.TargetExtraSearchRadius : WDist.Zero;
 
+					if (targetExtraSearchRadius == WDist.Zero)
```

Comment:
This will bypass the checks if one of the projectiles actually does set this to 0 (which is a legal value to set via yaml, but invalid to use ingame).